mmcv/ops/csrc/common/cuda/diff_iou_rotated_cuda_kernel.cuh

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) OpenMMLab. All rights reserved
+// Adapted from https://github.com/lilanxiao/Rotated_IoU/cuda_op/sort_vert_kernel.cu  # noqa
+#ifdef MMCV_USE_PARROTS
+#include "parrots_cuda_helper.hpp"
+#else
+#include "pytorch_cuda_helper.hpp"
+#endif
+
+#define MAX_NUM_VERT_IDX 9
+#define INTERSECTION_OFFSET 8
+#define EPSILON 1e-8
```

Comment:
You can use `-FLT_MAX` instead.

Reply:
I'm not sure that we want to modify it. Even now there some corner cases with incorrect results because of numerical instability (when one vertex of the first is exactly on the edge of the second box). However it is almost not the case for training neural networks in real life.

Reply:
Sure, I see.